### PR TITLE
Molecule notification hooks migration

### PR DESCRIPTION
**Pull request by juanmaguitar**
- 🌎Online Demo → https://sui-components-molecule-notification-hooks-migration-tskrnwkblz.now.sh/workbench/molecule/notification/demo

### CODE REVIEW

File: components/molecule/notification/src/index.js [hunk 1]
```diff
@@ -33,131 +34,295 @@ const VARIATIONS = {
...
+  )
+
+  const handleClickClose = e => {
+    setShow(false)
   }
 
-  toggleShow = () => {
-    const show = !this.state.show
-    const {onClose, effect} = this.props
+  // const handleToggleShow = useCallback(() => {
```
Comment (midudev) [inline components/molecule/notification/src/index.js]: 👀

File: components/molecule/notification/src/index.js [hunk 2]
```diff
@@ -33,131 +34,295 @@ const VARIATIONS = {
...
-      : this.setState({show})
+  //   if (show) {
+  //     if (autoCloseTimeInSeconds) triggerAutoClose(autoCloseTimeInSeconds)
+  //   } else {
+  //     clearTimeout(this.autoCloseTimeout)
+  //     onClose()
+  //   }
+  // }, [effect, getAutoCloseTime, onClose, show, triggerAutoClose])
+
+  useEffect(() => {
```
Comment (midudev) [inline components/molecule/notification/src/index.js:85]: 🧹 console.log

File: components/molecule/notification/src/index.js (hunk 1 above)
Comment (juanmaguitar) [reply components/molecule/notification/src/index.js]: Fixed! 👍

File: components/molecule/notification/src/index.js (hunk 2 above)
Comment (juanmaguitar) [reply components/molecule/notification/src/index.js:85]: Fixed! 👍

**Review by kikoruiz — APPROVED** [review]

**Review by midudev — APPROVED** [review]

**Pull request merged**